README.md

```diff
@@ -15,8 +15,10 @@
 All of the completions are [customizable](#customization).
 
 ## Requirements
-* PowerShell >= 5.0
 * docker cli >= 1.13 in `$env:PATH`
+* `$PSVersionTable.PSVersion -ge 5` for command support. 
+* One dash (`-`) and double dash (`--`) completers are not invoked in `PSVersion` of `5.*`. Type a character following `-` and `--`. For example, type `--a` and press tab.
+* `$PSVersionTable.PSVersion -ge 6` for completers with one `-` and double `--` dash without workarounds.
```

Comment:
These new lines can not be accepted. They are not requirements information. Additionally, the expression of version, `$PSVersionTable.PSVersion -ge 5`, reduces readability.